skbio/io/_registry.py

```diff
@@ -195,7 +177,7 @@ def decorator(reader):
         # We wrap the reader so that basic file handling can be managed
         # externally from the business logic.
         if cls is None:
-            def wrapped_reader(fp, mode='U', mutate_fh=False, **kwargs):
+            def wrapped_reader(fp, mode='r', mutate_fh=False, **kwargs):
```

Comment:
You should be able to drop mode here as well. (This might cause some test failures, but these tests can be reframed in the context of a registered operation rather than user input)